package.json

```diff
@@ -37,6 +38,7 @@
         "extensions": [
           ".fish"
         ],
+        "firstLine": "^#!(/usr)?(/bin/)(env[ \\t]+)?fish$",
```

Comment:
I like this idea, but I feel this is a little too specific. For instance, on my Mac, fish is actually located at `/usr/local/bin/fish`. And sometimes I have a local build of fish running at `/home/brian/bin/fish`.

How about we relax this condition to be "any shebang with the word `fish`"? This is what VSCode's built-in languages of node, ruby, python, and php all do:

```json
"firstLine": "^#!\\s*/.*\\bruby\\b",
```

https://github.com/microsoft/vscode/blob/c898cc34899bb3b3d5e2bcae881ae9a3f5fa3162/extensions/ruby/package.json#L18

So could this line be changed to this?

```json
"firstLine": "^#!\\s*/.*\\bfish\\b",
```